src/xdocs/config_javadoc.xml

```diff
@@ -233,6 +233,146 @@ public class TestClass {
       </subsection>
     </section>
 
+    <section name="JavadocBlockTagLocation">
+      <p>Since Checkstyle 8.24</p>
+      <subsection name="Description" id="JavadocBlockTagLocation_Description">
+        <p>
+          Checks that a
+          <a href="https://docs.oracle.com/en/java/javase/11/docs/specs/doc-comment-spec.html#block-tags">
+          javadoc block tag</a> appears only at the beginning of a line, ignoring leading asterisks
+          and white space.
+        </p>
+        <p>
+          Rationale: all javadoc block tags not placed at the beginning of a line
+          are treated as the plain text. This may be intentional, but changing
+          the text may unexpectedly place them at the beginning of the line and
+          they will no longer be recognized as plain text.
+          Conversely, if a block tag is accidentally moved to the middle of a line,
+          it will be treated as plain text without any violations, but the generated
+          documentation will be erroneous.
+        </p>
+      </subsection>
+      <subsection name="Notes"  id="JavadocBlockTagLocation_Notes">
+        <p>
+          To place a tag explicitly as text, escape the <code>@</code> symbol
+          with &amp;#64; or place it inside <code>{@code }</code>, for example:
+        </p>
+        <source>
+/**
+ * &amp;#64;serial literal in {&#64;code &#64;serial} Javadoc tag.
+ */
+        </source>
+      </subsection>
+      <subsection name="Properties" id="JavadocBlockTagLocation_Properties">
+        <table>
+          <tr>
+            <th>name</th>
+            <th>description</th>
+            <th>type</th>
+            <th>default value</th>
+            <th>since</th>
+          </tr>
+          <tr>
+            <td>tags</td>
+            <td>Specify the javadoc tags to process.</td>
+            <td><a href="property_types.html#stringSet">String Set</a></td>
+            <td><code>exception, hidden, author, deprecated, throws, version, see,
+              serialData, param, serial, provides, serialField, uses, return, since
```

Comment:
can we sort this list ?

Reply:
> can we sort this list ?

no. `com.puppycrawl.tools.checkstyle.internal.XdocsPagesTest.validatePropertySectionPropertyEx` wants it exactly in this order:
```
[ERROR] testAllCheckSections(com.puppycrawl.tools.checkstyle.internal.XdocsPagesTest)  Time elapsed: 0.02 s  <<< FAILURE!
org.junit.ComparisonFailure: config_javadoc.xml section 'JavadocBlockTagLocation' should have the value for tags expected:<[exception, hidden, author], deprecated, throws...> but was:<[author, exception, hidden], deprecated, throws...>
```

I'll recheck this. It looks like a bug.

Reply:
The values are coming from the `Set` for the field `tags`. The only way we can get these values is to iterate over the collection. Whatever the collection returns is the order of the items we go by. We don't do any extra sorting or such on that data.
See https://github.com/checkstyle/checkstyle/blob/765137a28f1bd4acb2294b8446c44d1828233934/src/test/java/com/puppycrawl/tools/checkstyle/internal/XdocsPagesTest.java#L1066

Reply:
done